index.js

```diff
@@ -120,7 +120,7 @@ ipc.send = async o => {
 
   let s = new URLSearchParams({
     event: o.event,
-    index: o.window || '0',
+    index: o.window ?? '-1',
```

Comment:
why -1?

Reply:
when it's `-1` we broadcast to all the windows

Reply:
oh nice!